pkg/minishift/openshift/version/openshift_versions_test.go

```diff
@@ -133,3 +133,25 @@ func Test_validate_openshift_min_versions(t *testing.T) {
 		}
 	}
 }
+
+func TestIsGreaterOrEqualToBaseVersion(t *testing.T) {
+	var versionTestData = []struct {
+		openshiftVersion     string
+		baseOpenshiftVersion string
+		expectedResult       bool
+	}{
+		{"v3.6.0", "v3.7.0", false},
+		{"v3.6.0-alpha.1", "v3.6.0-alpha.2", false},
+		{"v3.7.1", "v3.7.1", true},
+		{"v3.6.0-rc1", "v3.6.0-alpha.1", true},
+		{"v3.6.0-alpha.1", "v3.6.0-beta.0", false},
+	}
+
+	for _, versionTest := range versionTestData {
+		actualResult, _ := IsGreaterOrEqualToBaseVersion(versionTest.openshiftVersion, versionTest.baseOpenshiftVersion)
+		if actualResult != versionTest.expectedResult {
+			t.Errorf("IsGreaterOrEqualToBaseVersion(%s, %s) is expected to return '%v' but returned '%v'",
+				versionTest.openshiftVersion, versionTest.baseOpenshiftVersion, versionTest.expectedResult, actualResult)
+		}
+	}
+}
```

Comment:
Test for GreaterOrEqualToBaseVersion already exist and also have good test table along with error handle https://github.com/minishift/minishift/blob/master/pkg/minishift/openshift/version/openshift_versions_test.go#L100-L135 

I think we can just close this PR and relevant issue.

Reply:
@praveenkumar Also as discussed I have removed https://github.com/minishift/minishift/blob/master/pkg/minishift/openshift/version/openshift_versions_test.go#L100-L135 as we have https://github.com/minishift/minishift/blob/master/pkg/minishift/openshift/version/openshift_versions_test.go#L30-L65